docs/configuration_reference/model_loading.rst

```diff
@@ -46,3 +46,6 @@ preload_from_files
           }
         }
 
+init_missing_vars
+    If set to ``True``, any variables not found in the checkpoint will be randomly initialized.
```

Comment:
This is misleading. This is always the case.

It's just that we will throw an error when we do not found expected variables.